package.json

```diff
@@ -36,22 +37,37 @@
   "author": "GitHub",
   "license": "MIT",
   "devDependencies": {
+    "aws-sdk": "2.220.1",
```

Comment:
Looks like this may have snuck in from the previous S3 based solution?

Reply:
Yea that was a prereq for s3, should be removable.